setup.cfg

```
@@ -1,9 +1,10 @@
 [flake8]
-max-line-length = 119
-max-complexity = 18
-exclude =.git,__pycache__,docs/source/conf.py,build,dist
-ignore = I101,I201,F401,F403,S001,D100,D101,D102,D103,D104,D105,D106,D107,D200,D205,D400,W504,D202,E203,E501,E722,W503,B006
-inline-quotes = "
+max-line-length = 127
```

120 would be better I believe
```suggestion
max-line-length = 120
```